packages/dd-trace/test/opentracing/span.spec.js

```diff
@@ -470,5 +471,40 @@ describe('Span', () => {
 
       expect(processor.process).to.have.been.calledOnce
     })
+
+    describe('tracePropagationBehaviorExtract and Baggage', () => {
+      let parent
+
+      beforeEach(() => {
+        parent = {
+          traceId: '123',
+          spanId: '456',
+          _baggageItems: {
+            foo: 'bar'
+          },
+          _trace: {
+            started: ['span'],
+            finished: ['span']
+          },
+          _isRemote: true
+        }
+      })
+
+      it('should not propagate baggage items when Trace_Propagation_Behavior_Extract is set to ignore', () => {
+        process.env.DD_TRACE_PROPAGATION_BEHAVIOR_EXTRACT = 'ignore'
+        const config = new Config()
+        const tracers = new DatadogTracer(config, undefined)
+        span = new Span(tracers, processor, prioritySampler, { operationName: 'operation', parent })
```

Comment:
what if instead of tracers you pass in parent? that way you won't need to import DatadogTracer